packages/edit-site/src/components/page-patterns/patterns-list.js

```diff
@@ -94,7 +96,9 @@ export default function PatternsList( { categoryId, type } ) {
 			{ ! isResolving && !! unsyncedPatterns.length && (
 				<>
 					<VStack className="edit-site-patterns__section-header">
-						<Heading level={ 4 }>{ __( 'Standard' ) }</Heading>
+						<Heading as="h2" level={ 4 }>
```

Comment:
`as` takes [precedence](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/heading/hook.ts#L19) over `level`. No need to supply both.